emailApp/sendMail/mail.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import {Icon} from 'react-native-elements';
+import Tts from 'react-native-tts';
+import {emailDataCleaning} from '../dataCleaning/cleaner';
+
+const Email = require('../smtp/smtp');
```

Comment:
```const Email = require('../smtp/smtp');
const keys = require('../keys/keys');```
either use small or capital at all places for "const names".

Reply:
done